networking_cisco/plugins/cisco/cfg_agent/device_drivers/asr1k/aci_asr1k_routing_driver.py

```diff
@@ -542,3 +538,38 @@ def _set_vrf_pid(self, ri, port, config):
         # TODO(tbachman): fix to ensure PID uniqueness
         pid = int(hashlib.md5(vrf).hexdigest()[:4], 16)
         return str(pid)
+
+    def _get_snat_pool_name(self, subnet):
+        is_multi_region_enabled = cfg.CONF.multi_region.enable_multi_region
```

Comment:
nit: since is_multi_region_enabled & region_id got used only once, can we just use cfg.CONF.multi_region.enable_multi_region & cfg.CONF.multi_region.region_id directly so the code will be more terse?